include/date/date.h

```diff
@@ -63,7 +63,16 @@
 #include <stdexcept>
 #include <string>
 #if HAS_STRING_VIEW
-# include <string_view>
+#  if defined(__has_include)
+#    if __has_include(<string_view>)
+#      include <string_view>
+#    elif __has_include(<experimental/string_view>)
+#      include <experimental/string_view>
+       namespace std {
+           typedef experimental::string_view string_view;
```

Comment:
Maybe it would be better to actually use C++17 features like `using string_view = experimental::string_view;` :P

Reply:
Totally agreed. I was mixing C and C++ by the time I wrote the code and I got confused. I fixed it, thank you!